.github/workflows/bump-version-release.yaml

```diff
@@ -1,101 +1,104 @@
-
-      - name: Tag and release
-        uses: bakdata/ci-templates/actions/tag-and-release@v1.22.0
-        with:
-          tag: "${{ steps.bump-version.outputs.release-version }}"
-          github-username: ${{ secrets.github-username }}
-          github-email: ${{ secrets.github-email }}
-          github-token: ${{ secrets.github-token }}
-          release-title: "${{ steps.bump-version.outputs.release-version }}"
-          release-body: "${{ steps.build-changelog.outputs.single-changelog }}"
+    release:
+        runs-on: ubuntu-22.04
+        outputs:
+            release-version: ${{ steps.bump-version.outputs.release-version }}
+            old-version: ${{ steps.bump-version.outputs.old-version }}
+        steps:
+        -   name: Authenticate with GCloud
+            uses: google-github-actions/auth@v2
+            id: auth
+            with:
+                workload_identity_provider: ${{ secrets.GOOGLE_WORKLOAD_IDENTITY_PROVIDER }}
+                service_account: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
+        -   name: Fetch secrets from GSM
+            id: fetch-secrets
+            uses: google-github-actions/get-secretmanager-secrets@v2
+            with:
+                secrets: |-
+                    github-email:${{ secrets.GOOGLE_PROJECT_ID }}/GH_EMAIL
+                    github-token:${{ secrets.GOOGLE_PROJECT_ID }}/GH_TOKEN
+                    github-username:${{ secrets.GOOGLE_PROJECT_ID }}/GH_USERNAME
+                export_to_environment: 'true'
+        -   name: Check out repository
+            uses: bakdata/ci-templates/actions/checkout@1.49.0
+            with:
+                ref: ${{ github.event.repository.default_branch }}
+                persist-credentials: false
+                fetch-depth: 0
+        -   name: Bump version
+            id: bump-version
+            uses: bakdata/ci-templates/actions/bump-version@v1.21.2
+            with:
+                release-type: ${{ inputs.release-type }}
+                working-directory: ${{ inputs.working-directory }}
+        -   name: Create changelog
+            id: build-changelog
+            uses: bakdata/ci-templates/actions/changelog-generate@1.52.1
+            if: ${{ inputs.changelog }}
+            with:
+                github-token: ${{ env.github-token }}
+                tag: ${{ steps.bump-version.outputs.release-version }}
+                changelog-file: ${{ inputs.changelog-file }}
+        -   name: Commit and push changes including .bumpversion.cfg file
+            uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+            with:
+                ref: ${{ github.event.repository.default_branch }}
+                commit-message: "Bump version ${{ steps.bump-version.outputs.old-version }} \u2192 ${{ steps.bump-version.outputs.release-version }}"
+                github-username: ${{ env.github-username }}
+                github-email: ${{ env.github-email }}
+                github-token: ${{ env.github-token }}
```

Comment:
These don't seem right if you check in GSM. It should be `${{ env.GH_USERNAME }}` now (similar for the other ones), right?

This is similar for all the other workflows that you changed.